drivers/homeless_summary_report/app/models/homeless_summary_report/report.rb

```diff
@@ -459,6 +459,8 @@ def title_for(household_category, demographic_category)
               report_client[:last_name] = hud_client.last_name
               report_client[:report_id] = id
               report_client["spm_#{spm_field}"] = parts.fetch(:value_accessor).call(spm_member)
+              # In Measure 7b, set the destination to 0 to flag the destination as 'Remained housed' for clients who have moved in
```

Comment:
Does this need to confirm they have a move-in date and their Exit is after the report end?  
Does it maybe also need to check for particular destinations that constitute permanent housing?

Reply:
yes, thank you! For whatever reason I was thinking building this off of a run SPM was taking care of this, but that's not always correct. I'll get something added to the PR to account for this

Reply:
Adding additional checks for flagging members as 'Remained Housed' in [this commit](https://github.com/greenriver/hmis-warehouse/pull/4905/commits/aca1d52aaf901da7fb077b9d419445f8c40ec4e5)